charts/opentelemetry-operator/templates/_helpers.tpl

```diff
@@ -117,9 +117,9 @@ a cert is loaded from an existing secret or is provided via `.Values`
 {{- $caCertEnc = b64enc $ca.Cert }}
 {{- end }}
 {{- else }}
-{{- $certCrtEnc = b64enc .Values.admissionWebhooks.cert_file }}
-{{- $certKeyEnc = b64enc .Values.admissionWebhooks.key_file }}
-{{- $caCertEnc = b64enc .Values.admissionWebhooks.ca_file }}
+{{- $certCrtEnc = .Files.Get .Values.admissionWebhooks.cert_file | b64enc }}
```

Comment:
I would prefer to rename these variables to be camelCase for consistency. Let me know if sounds okay.

Reply:
Yup